Add new term entry for the Python string method .center() (#3327)
Codecademy/docs: content/python/concepts/strings/terms/center/center.md

---
Title: '.center()'

> Description: 'Returns a new string with added specified padding. It's default is whitespace.'

Reviewer: ```suggestion
Description: 'Returns a new string with the specified padding.'
```

Subjects:
  - 'Data Science'  
  - 'Computer Science'  
Tags:
  - 'Methods'  
CatalogContent:
  - 'learn-python-3' 
  - 'paths/computer-science' 
--- 

> The **`.center()`** string method returns a new string that has been amended with added padding. The default padding is whitespace, but any character may be used, including numbers or symbols.

Reviewer: ```suggestion
The **`.center()`** string method returns a new string that has been amended with padding. The padding is allocated evenly to both sides of the string, depending on the overall length specified. The default padding is whitespace, but any character may be used, including numbers or symbols.
```

## Syntax

```pseudo
string.center(length, 'character')
```

> The `length` parameter is required and must be an integar. This will be the length of new string.

Reviewer: ```suggestion
The `length` parameter is required and must be an integer. This will be the length of new string.
```

The `.center()` method accepts an optional `'character'` parameter. If no `character` is specified the default is whitespace.

## Example

In the following example the `.center()` method is used to create a string that has a length of 10 characters. The padding character is a '+':

```python
str = "Hello"

> new_str = str.center(10, '+')

Reviewer: it may be better to se the length to 11 in this case so that there are an equivalent number of `+` on either side of the word
